_api-reference/snapshots/restore-snapshot.md

```diff
@@ -45,6 +45,7 @@ All request body parameters are optional.
 | partial | Boolean | How the restore operation will behave if indices in the snapshot do not have all primary shards available. If `false`, the entire restore operation fails if any indices in the snapshot do not have all primary shards available. <br /> <br />If `true`, allows the restoration of a partial snapshot of indices with unavailable shards. Only shards that were successfully included in the snapshot are restored. All missing shards are recreated as empty. By default, the entire restore operation fails if one or more indices included in the snapshot do not have all primary shards available. To change this behavior, set `partial` to `true`. Defaults to `false`. |
 | rename_pattern | String | The pattern to apply to restored data streams and indices. Data streams and indices matching the rename pattern will be renamed according to `rename_replacement`. <br /><br />The rename pattern is applied as defined by the regular expression that supports referencing the original text. <br /> <br />The request fails if two or more data streams or indices are renamed into the same name. If you rename a restored data stream, its backing indices are also renamed. For example, if you rename the logs data stream to `recovered-logs`, the backing index `.ds-logs-1` is renamed to `.ds-recovered-logs-1`. <br /> <br />If you rename a restored stream, ensure an index template matches the new stream name. If there are no matching index template names, the stream cannot roll over and new backing indices are not created.|
 | rename_replacement | String | The rename replacement string. See `rename_pattern` for more information.|
+| source_remote_store_repository | String | The name of the remote store repository of the source index that is being restored. If not provided, the repository that was registered when the snapshot was created will be used.
```

Comment:
Would the second sentence be better (more active) as "If not provided, [noun] will use the repository that was registered when the snapshot was created"?

Reply:
Snapshot restore API